### ISSUE TITLE
Dissonance For Mirror Networking is not the latest version in asset store

### ISSUE THREAD

**Issue by shiena** [Placeholder-Software/Dissonance#150]
Currently, Dissonance For Mirror Networking is the latest version 6.3.6, but the asset store version is 6.3.1. This version does not work with the latest Mirror in the asset store.
https://github.com/Placeholder-Software/Dissonance/issues/146#issuecomment-480316318

Are you not distributing version 6.3.6 in the asset store?

**Comment by martindevans** [comment]
Sorry about this, there's been a bit of confusion about the versioning with Mirror+Dissonance. Traditionally the version of an integration is the version of the minimum Dissonance version it works with. So we initially released `Dissonance-MirrorIgnorance-6.3.1` with Dissonance 6.3.1, however there were then multiple changed versions of Mirror during the time that Dissonance 6.3.1 was the latest version so we ended up with a slightly odd versioning scheme (bumping the minor version of the integration). It's all a bit confusing!

I've just double checked that the version on the store is the correct version - it's equivalent to the `Dissonance-MirrorIgnorance-6.3.6` package previously released [here](https://placeholder-software.co.uk/dissonance/releases/6.3.1.html).

I imported that into a project with Dissonance and Mirror from the store and it looks like it all works - what problem did you encounter?
